crmsh/bootstrap.py

```diff
@@ -856,27 +866,18 @@ def pick_default_value(vlist, ilist):
     mcastport_res = []
     default_ports = ["5405", "5407"]
     two_rings = False
-    default_networks = []
 
-    if _context.ipv6:
-        network_list = []
-        all_ = utils.network_v6_all()
-        for item in all_.values():
-            network_list.extend(item)
-        default_networks = map(utils.get_ipv6_network, network_list)
-    else:
-        network_list = utils.network_all()
-        if len(network_list) > 1:
-            default_networks = [_context.ip_network, network_list.remove(_context.ip_network)]
-        else:
-            default_networks = [_context.ip_network]
-    if not default_networks:
+    local_iplist = utils.ip_in_local(_context.ipv6)
+    len_iplist = len(local_iplist)
+    if len_iplist == 0:
         error("No network configured at {}!".format(utils.this_node()))
 
+    default_ip = [_context.ip_address] + [ip for ip in local_iplist if ip != _context.ip_address]
```

Comment:
I don't understand this... You are avoiding the `_context.ip_address` in the loop to add it later...
If they idea is to remove duplicates better something like: `list(dict.fromkeys(mylist))`

And maybe change `default_ip` by `current_ips` as the variable is a list with the current addresses, default looks like not the meaning of this list

Reply:
Or simply add `_context.ip_address` if it's not in the list. Otherwise is hard to undestand the meaning of this code line

Reply:
The logic here, is put `_context.ip_address` the **first place** to pick as default value.
`_context.ip_address` came from `ip -o route show`, may have higher priority;
While `local_iplist` came from `ip addr show`, maybe include lots of addresses but not very useful(maybe unreachable, I guess)

Reply:
Ok, now I understand, It would be nice to put this as a comment in the top of the line (just saying that this operation is used to put in the 1st place)